src/OpenTelemetry/Trace/ActivityExporter.cs

```diff
@@ -62,12 +62,23 @@ public abstract class ActivityExporter : IDisposable
         /// <returns>Returns <see cref="Task"/>.</returns>
         public abstract Task ShutdownAsync(CancellationToken cancellationToken);
 
+        /// <summary>
+        /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
+        /// </summary>
         public void Dispose()
         {
             this.Dispose(true);
             GC.SuppressFinalize(this);
         }
 
+        /// <summary>
```

Comment:
This is the standard text I use for `protected virtual void Dispose` taken off one of the methods in .NET:
```csharp
		/// <summary>
		/// Releases the unmanaged resources used by this class and optionally releases the managed resources.
		/// </summary>
		/// <param name="isDisposing"><see langword="true"/> to release both managed and unmanaged resources; <see langword="false"/> to release only unmanaged resources.</param>
```
May or may not want to use that, a bit simpler 🤷

Reply:
just updated :)